components/gallery/GalleryItem.vue

```diff
@@ -107,4 +113,8 @@ const CarouselTypeVisited = defineAsyncComponent(
     margin-top: 1.25rem;
   }
 }
+
+hr {
+  background: $k-grey;
+}
```

Comment:
Same stuff as https://github.com/kodadot/nft-gallery/pull/4582#discussion_r1059876340

Reply:
This was removed and is resolved globally.